src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -74,22 +74,27 @@ function calculateAudioScore(
     return roundTripTime;
   };
 
+  const getDelay = (): number => {
+    const roundTripTime = getRoundTripTime();
+    const delay = (roundTripTime * MS_PER_SEC) / 2;
+    return delay || DEFAULT_DELAY;
+  };
+
   const audioScore = (packetLossRatio: number) => {
-    const LOCAL_DELAY = 20; // 20 msecs: typical frame duration
+    const LOCAL_DELAY = 30; // 30 msecs: typical frame duration
```

Comment:
Changed the local delay based on https://github.com/opentok/gru/blob/develop/src/main/java/com/tokbox/gru/extractors/AudioScoreAlgorithm.java#L14